### PR TITLE
FS-1163 Add authentication util

### PR DESCRIPTION
**Pull request by harryyo**
Implement @login_required authentication util from fsd_utils to protect routes

### CODE REVIEW

File: config/envs/development.py [hunk 1]
```diff
@@ -12,3 +12,18 @@ class DevelopmentConfig(DefaultConfig):
 
     FSD_LOGGING_LEVEL = logging.DEBUG
     USE_LOCAL_DATA = strtobool(getenv("USE_LOCAL_DATA", "True"))
+
+    COOKIE_DOMAIN = getenv("COOKIE_DOMAIN", ".localhost.localhost")
```
Comment (gidsg) [inline config/envs/development.py:16]: Is `.localhost.localhost` the correct value for local development? Or is it OK for this to be None?

Comment (harryyo) [reply config/envs/development.py]: Actually unused here though I think! Removed

File: requirements.txt [hunk 2]
```diff
@@ -16,51 +16,67 @@ click==8.1.3
...
+flask-compress==1.10.1
     # via -r requirements.in
-flask-talisman==1.0.0
+flask-talisman==0.8.1
     # via -r requirements.in
-flask-wtf==1.0.1
+flask-wtf==1.0.0
     # via -r requirements.in
-funding-service-design-utils @ https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.8.tar.gz
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils.git@fs-1163-add-security-util
```
[the remark this answers is not in the document]
Comment (harryyo) [reply requirements.txt:43]: Yes, pending new utils release with the security branch updates - will update when that's merged

**Review by gidsg — APPROVED** [review]
happy to merge once pointing to released version of utils.

File: requirements.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (harryyo) [reply requirements.txt]: Updated to 0.0.9

**Pull request merged**